src/roles/vnsutil-deploy/tasks/openstack.yml

```diff
@@ -0,0 +1,58 @@
+- name: Set local variable with upgrade status
+  set_fact:
+    upgrade: "{{ nuage_upgrade|default('False') }}"
+
+- block:
+  - name: Verify that upgrade_vmname is defined if this is an upgrade
```

Comment:
We don't support upgrade... This should not be here...

Reply:
I forgot that was still in there, I removed it now, sorry about that